From 59e0d935effe4b20e81342fe0d1f5d1070b1c80c Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Wed, 14 Dec 2022 22:22:43 +0100
Subject: [PATCH] plt: Always load the legacy dtb

We enabled dm for i2c leds and this causes the
eeprom that contains the hw revision to not get
loaded correctly, but we know the hw variant for
this device type so let's always load the legacy dtb.

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/imx8mm_var_dart.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx8mm_var_dart.h b/include/configs/imx8mm_var_dart.h
index dd24fb38ca..9b3dfb255e 100644
--- a/include/configs/imx8mm_var_dart.h
+++ b/include/configs/imx8mm_var_dart.h
@@ -140,7 +140,7 @@
 					"if test $carrier_rev = legacy; then " \
 						"setenv fdt_file imx8mm-var-dart-dt8mcustomboard-legacy.dtb; " \
 					"else " \
-						"setenv fdt_file imx8mm-var-dart-dt8mcustomboard.dtb; " \
+						"setenv fdt_file imx8mm-var-dart-dt8mcustomboard-legacy.dtb; " \
 					"fi; " \
 				"fi; " \
 			"fi; " \
-- 
2.37.2

